tasks/compile-example.js

```diff
@@ -57,14 +57,21 @@ module.exports = function(templateRoot, template) {
       const document = DocumentParser.parse(contents);
       const stream = this;
       const title = FileName.toString(file);
+      let next = FileName.nextFile(file, 'src');
+      if (next) {
+        next = FileName.toString(next) + '.html';
```

Comment:
I got some undefined at some point and the toString doesn't handle that correctly.